flask_restful/reqparse.py

```diff
@@ -270,11 +291,16 @@ def parse_args(self, req=None, strict=False):
         # A record of arguments not yet parsed; as each is found
         # among self.args, it will be popped out
         req.unparsed_arguments = dict(self.argument_class('').source(req)) if strict else {}
-
+        errors = {}
         for arg in self.args:
-            value, found = arg.parse(req)
+            value, found = arg.parse(req, self.bundle_errors)
+            if isinstance(value, ValueError):
+                errors.update(found)
+                found = None
             if found or arg.store_missing:
                 namespace[arg.dest or arg.name] = value
+        if errors:
+            flask_restful.abort(400, message=json.dumps(errors))
```

Comment:
@alexmorken Why you did json.dumps here? In a client i got string instead json.
I've created pull request for you https://github.com/alexmorken/flask-restful/pull/1